icu4c/source/data/mappings/gb18030.ucm

```diff
@@ -28077,6 +28077,22 @@ CHARMAP
 <U9FA3> \xFD\x99 |0
 <U9FA4> \xFD\x9A |0
 <U9FA5> \xFD\x9B |0
+<U9FB4> \xFE\x59 |0
+<U9FB4> \x82\x35\x90\x37 |3
```

Comment:
what does the "|3" in the third field mean? why is it "|3" ?

Reply:
See https://unicode-org.github.io/icu/userguide/conversion/data.html#ucm-file-format. It is a fallback mapping only from GB18030 to Unicode , not the other way.